website/cue/reference/urls.cue

```diff
@@ -383,6 +383,7 @@ urls: {
 	nixos:                                      "https://nixos.org/"
 	nixpkgs_9682:                               "\(github)/NixOS/nixpkgs/issues/9682"
 	openssl:                                    "https://www.openssl.org/"
+	openssl_conf:                               "https://www.openssl.org/docs/manmaster/man5/config.html"
```

Comment:
Same here regarding using a concrete version number for the upstream documentation:

<https://www.openssl.org/docs/man3.1/man5/config.html>